app/controllers/posts_controller.rb

```diff
@@ -1,43 +1,35 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @pagy, @posts = pagy(@user.posts)
+    @post = Post.where(author_id: params[:user_id])
+
+    render '/posts/index'
```

Comment:
- [ ] **[OPTIONAL]** I don't see the point of having this `render '/posts/index'` path here. There is a post index page in the views that takes care of the render automatically, I no point of doing this here. It could also contribute to why your tests are failing.